src/test/java/org/jenkinsci/plugins/workflow/job/WorkflowRunTest.java

```diff
@@ -473,4 +479,17 @@ private void assertCulprits(WorkflowRun b, String... expectedIds) throws IOExcep
         assertFalse(b.isLogUpdated());
     }
 
+    @Test public void getScms() throws Exception {
+        sampleRepo.init();
+        sampleRepo.write("Jenkinsfile", "echo 'hello'");
+        sampleRepo.git("add", "Jenkinsfile");
+        sampleRepo.git("commit", "--message=files");
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        CpsScmFlowDefinition def = new CpsScmFlowDefinition(new GitStep(sampleRepo.toString()).createSCM(), "Jenkinsfile");
+        p.setDefinition(def);
+        WorkflowRun b = r.buildAndAssertSuccess(p);
+        List<SCM> checkouts = b.getCheckouts();
+        assertTrue(checkouts.size() == 1);
+        assertTrue(checkouts.get(0).getClass().getName().equals(GitSCM.class.getName()));
```

Comment:
nit: Always better to use `assertEquals` or `assertThat` for better messages if the assertion fails.

```suggestion
        assertEquals(GitSCM.class, checkouts.get(0).getClass());
```

Reply:
it had been a while that i forgot which one was the preferred route :p

Reply:
No worries, with `assertTrue`/`assertFalse`, the failure just says "assertion failed", but with `assertEquals`, the message will show you the expected and actual values, which is much more useful. If all you have is a boolean value, `assertTrue`/`assertFalse` are fine, but if you are comparing values, better to use `assertEquals` or `assertThat` for complex comparisons.